app/(dashboard)/mailboxes/[mailbox_slug]/saved-replies/page.tsx

```
@@ -46,21 +46,21 @@ export default function SavedRepliesPage() {
-    toast({ title: "Saved reply updated successfully", variant: "success" });
+    showSuccessToast("Saved reply updated successfully");
   };
 
   const handleCopySavedReply = async (content: string) => {
     try {
       await navigator.clipboard.writeText(content);
-      toast({ title: "Saved reply copied to clipboard", variant: "success" });
-    } catch (_error) {
-      toast({ title: "Failed to copy saved reply", variant: "destructive" });
+      showSuccessToast("Saved reply copied to clipboard");
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)